Sofa/GUI/Component/src/sofa/gui/component/performer/RemovePrimitivePerformer.inl

```diff
@@ -370,10 +370,11 @@ bool RemovePrimitivePerformer<DataTypes>::createElementList()
             for(unsigned int i=0; i<listObject.size(); ++i) // loop on all components to find mapping (only tetra for the moment)
             {
                 sofa::core::topology::TopologicalMapping *topoMap = dynamic_cast<sofa::core::topology::TopologicalMapping *>(listObject[i]);
+                const sofa::type::vector<Index>& topoMapIndices = topoMap->Loc2GlobDataVec.getValue();
                 if (topoMap)
                 {
                     // Mapping found: 1- get surface element ID in volumique topology, 2- get volume element ID behind surface element, 3- switching all variables to volumique case
-                    unsigned int volTmp = (topoMap->getLoc2GlobVec()).getValue()[selectedElem[0]];
+                    Index volTmp = topoMapIndices[selectedElem[0]];
```

Comment:
Looks like this change does not belong to this PR

Reply:
![Indeed](https://tenor.com/bnww1.gif)